js/src/hooks/useNotices.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { STORE_KEY } from '.~/data/constants';
+
+/**
+ * @typedef {import('@wordpress/notices').Notice} Notice
+ */
+
+/**
+ * A hook that returns the WP notices
+ *
+ * @param {string} [storeKey] The store key
+ *
+ * @return {Array<Notice>} Returns the Notices
+ */
+const useNotices = ( storeKey = STORE_KEY ) => {
```

Comment:
❓ Similar concern to https://github.com/woocommerce/google-listings-and-ads/pull/1564#discussion_r898657626.